<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightItemWidget</class>
 <widget class="QWidget" name="FlightItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
  <property name="geometry">
   <rect>
    <x>10</x>
    <y>30</y>
    <width>580</width>
    <height>151</height>
   </rect>
  </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #fbfffe;</string>
   </property>
   <widget class="QPushButton" name="bookBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #156080;
    color: white;
    border: 1px solid #ababab;
    border-radius: 12px;
}</string>
    </property>
    <property name="text">
     <string>预订</string>
    </property>
   </widget>
   <widget class="QPushButton" name="collectBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e98532;
    color: white;
    border: 1px solid #ababab;
    border-radius: 12px;
}</string>
    </property>
    <property name="text">
     <string>收藏</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>130</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>起飞时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="takeoffTimeLabel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>2026年1月1日10：30</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>出发地</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="departureLabel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>北京大兴机场</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>130</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>到达时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="arriveTimeLabel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>2026年1月1日15:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>目的地</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="destinationLabel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>广州白云机场</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>110</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>航班号</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="airlineLogoLabel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="figures.qrc">:/img/source/figures/capitalLogo.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="flightNoLabel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>国航CA8394</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>40</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border：none；</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="figures.qrc">:/img/source/figures/arrow.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>90</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>剩余座位数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="remainingLabel">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>51</red>
             <green>182</green>
             <blue>15</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>51</red>
             <green>182</green>
             <blue>15</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>251</red>
             <green>255</green>
             <blue>254</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>33/80</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>票价</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="priceLabel">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>455元</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="figures.qrc"/>
 </resources>
 <connections/>
</ui>
